### PR TITLE
refactor(logging-apps): logging-apps 0.13.1

### PR DESCRIPTION
**Pull request by in0rdr**
* bump logging-apps appVersion to 0.14.0
* bump to fluentbit targetRevision 0.19.*
* fix fluentbit name
* fix template name

# Description

This is a review of https://github.com/adfinis-sygroup/helm-charts/pull/392

# Issues

There were minor issues while creating the fluentbit app with Argo.

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/logging-apps/values.yaml [hunk 1]
```diff
@@ -58,22 +58,22 @@ fluentd:
...
 
-# fluentBit -- [fluentBit](https://github.com/fluent/fluent-bit) ([example](./examples/fluentBit.yaml))
+# fluentbit -- [fluentbit](https://github.com/fluent/fluent-bit) ([example](./examples/fluentbit.yaml))
 # @default -- -
-fluentBit:
-  # fluentBit.enabled -- Enable fluentBit
+fluentbit:
+  # fluentbit.enabled -- Enable fluentbit
   enabled: false
-  name: fluentBit
```
Comment (eyenx) [inline charts/logging-apps/values.yaml:66]: This might be the sole Issue.

File: charts/logging-apps/templates/fluentbit.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-{{ if .Values.fluentBit.enabled }}
-{{ template "argoconfig.application" (list . "logging-apps.fluent") }}
```
Comment (eyenx) [inline charts/logging-apps/templates/fluentbit.yaml:2]: Exactly here. This should be fluentBit to function properly.

File: charts/logging-apps/values.yaml [hunk 3]
```diff
@@ -73,7 +73,7 @@ fluentBit:
   # fluentBit.chart -- Chart
   chart: "fluent-bit"
   # fluentBit.targetRevision -- [fluent-bit Helm chart](https://github.com/fluent/helm-charts/tree/main/charts/fluent-bit) version
-  targetRevision: "0.17.*"
+  targetRevision: "0.19.*"
```
[the remark this answers is not in the document]
Comment (in0rdr) [reply charts/logging-apps/values.yaml:76]: thank you, I just added the change (incl. pre-commit)

**Review by eyenx — APPROVED** [review]

**Review by hairmare — APPROVED** [review]

**Comment by hairmare** [comment]
nice fix!

**Pull request merged**